RELEASE.md

```diff
@@ -1,3 +1,11 @@
+# Release 1.13.1
+
+## Bug fixes
+- Fix launch error on Windows (#1900)
+- Fix What-If Tool loading examples without inference (#1898)
+- Disable the profile dashboard inside Colab, where it doesn’t work (#1914, #1922, #1945)
```

Comment:
I've been wrapping to 80 chars except for long URLs per the internal markdown style.

Reply:
Done.